:book: add information about filling in the fields (#158)
itpp-labs/sync-addons: openapi/doc/index.rst

  -d '{ "args": [[["is_company", "=", "True" ]]]}'

> Fill in field in existing record

Reviewer: ```suggestion
Updating existing record
```

-----------------------------

> We can do so using PUT method like this:

Reviewer: ```suggestion
For example, to set *phone* value for a partner, make a PUT request in the following way:
```

.. code-block:: bash

  curl -X PUT -H "Authorization: Basic BASE64_ENCODED_EXPRESSION" \
  -H "Content-Type: application/json" -H "Accept: */*" \
  -d '{ "phone": "+7123456789"}' "http://example.com/api/v1/demo/res.partner/41"

> Or if you need to fill in many2one field you can do so:

Reviewer: ```suggestion
To set many2one field, you need to pass id as a value:
```

.. code-block:: bash

  curl -X PUT -H "Authorization: Basic BASE64_ENCODED_EXPRESSION" \
  -H "Content-Type: application/json" -H "Accept: */*" \

>   -d '{ "title": *RECORD_ID*}' "http://example.com/api/v1/demo/res.partner/41"

Reviewer: титл это не many2one поле. Сделай company_id или parent_id
